docs/source/get_started/install.md

```diff
@@ -13,54 +13,15 @@
 
 ## Create a virtual environment for your Kedro project
 
-We strongly recommend [installing `conda` as your virtual environment manager](https://docs.conda.io/projects/conda/en/latest/user-guide/install/) if you don't already use it.
+We strongly recommend utilizing `venv` as your virtual environment manager if you don't already use it.
 
 ``` {tip}
 [Read more about virtual environments for Python projects](https://realpython.com/python-virtual-environments-a-primer/) or [watch an explainer video about them](https://youtu.be/YKfAwIItO7M).
 ```
 
-### How to create a new virtual environment using `conda`
+### How to create a new virtual environment using `venv`
 
-The recommended approach. From your terminal:
-
-```bash
-conda create --name kedro-environment python=3.10 -y
-```
-
-The example below uses Python 3.10, and creates a virtual environment called `kedro-environment`. You can opt for a different version of Python (any version >= 3.8 and <3.12) for your project, and you can name it anything you choose.
-
-The `conda` virtual environment is not dependent on your current working directory and can be activated from any directory:
-
-```bash
-conda activate kedro-environment
-```
-
-To confirm that a valid version of Python is installed in your virtual environment, type the following in your terminal (macOS and Linux):
-
-```bash
-python3 --version
-```
-
-On Windows:
-
-```bash
-python --version
-```
-
-To exit `kedro-environment`:
-
-```bash
-conda deactivate
-```
-
-### How to create a new virtual environment without using `conda`
-
-Depending on your preferred Python installation, you can create virtual environments to work with Kedro using `venv` or `pipenv` instead of `conda`.
-
-<details>
-<summary><b>Click to expand instructions for <code>venv</code></b></summary>
-
-If you use Python 3, you should already have the `venv` module installed with the standard library. Create a directory for working with your project and navigate to it. For example:
+The recommended approach. If you use Python 3, you should already have the `venv` module installed with the standard library. Create a directory for working with your project and navigate to it. For example:
```

Comment:
Is it missing some venv activate script? I expect something like `bin/activate`

Reply:
it is provided, I haven't change that part:
```
Activate this virtual environment:
source .venv/bin/activate # macOS / Linux
```